kong/plugins/proxy-cache/cache_key.lua

```diff
@@ -1,12 +1,13 @@
 local fmt = string.format
 local ipairs = ipairs
-local md5 = ngx.md5
 local type = type
 local pairs = pairs
 local sort = table.sort
 local insert = table.insert
 local concat = table.concat
 
+local sha256_hex = require "kong.tools.utils".sha256_hex
```

Comment:
Here we can define a more common name like `digest_func` to cover the changes of md5 and sha256.

Reply:
this name is intentionally specific to ensure the implementation (hash algorithm used) can be more easily recognized through the code